src/components/Snackbar/useSnackbar.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2024 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useContext } from 'react';
+import { SnackbarContext, SnackbarContextValue } from './SnackbarProvider';
+
+export function useSnackbar(): SnackbarContextValue {
+  const value = useContext(SnackbarContext);
+
+  if (!value) {
+    throw new Error(
+      'useImageUpload can only be used inside of <SnackbarProvider>',
```

Comment:
typo, right? should be `useSnackbar can only be used...`?

Reply:
Copy :cat2: alert :wink: